cicecore/cicedynB/analysis/ice_history.F90

```diff
@@ -3925,17 +4399,28 @@ subroutine accum_hist (dt)
       ! snapshots
       !---------------------------------------------------------------
 
-          ! compute sig1 and sig2
-        
-           call principal_stress (nx_block,  ny_block,  &
-                                  stressp_1 (:,:,iblk), &
-                                  stressm_1 (:,:,iblk), &
-                                  stress12_1(:,:,iblk), &
-                                  strength  (:,:,iblk), &
-                                  sig1      (:,:,iblk), &
-                                  sig2      (:,:,iblk), &
-                                  sigP      (:,:,iblk))
- 
+            ! compute sig1 and sig2
+            select case (grid_ice)
+            case('B')
+               call principal_stress (nx_block,  ny_block,  &
```

Comment:
For now, I am calling principal_stress with the stressp, stressm and stress12 values at the T points. This should probably be at the U points to be consistent with the B-grid. However, I think it can wait.